source/php-libraries.txt

```diff
@@ -12,133 +12,83 @@ PHP Libraries, Frameworks, and Tools
    :depth: 2
    :class: singlecol
 
-Libraries for the ``mongodb`` Extension
----------------------------------------
+Standalone Libraries
+--------------------
 
-Stand-alone Libraries
-~~~~~~~~~~~~~~~~~~~~~
+- `Doctrine MongoDB ODM <https://github.com/doctrine/mongodb-odm>`__ is a
+  library that provides object mapping functionality for MongoDB.
```

Comment:
S: [compound modifiers](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers)

```suggestion
  library that provides object-mapping functionality for MongoDB.
```